### PR TITLE
css for header and homepage

### PR DESCRIPTION
**Pull request by angelawong3**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -13,13 +20,8 @@
     "react-dom": "^17.0.1",
     "react-router-dom": "^6.2.1",
     "react-scripts": "4.0.1",
-    "web-vitals": "^0.2.4",
```
Comment (mohamedisaaq0) [inline client/package.json:16]: why are these dependencies removed?

Comment (angelawong3) [reply client/package.json:16]: i dont know why...i didn't edit the package.json file

Comment (angelawong3) [reply client/package.json:16]: they are not deleted..i added the following dependencies, the location of the other dependencies moved
"@fortawesome/free-solid-svg-icons": "^6.2.0",
 "@fortawesome/react-fontawesome": "^0.2.0",
 
You can see all the dependencies when you view the whole file

Comment (mohamedisaaq0) [reply client/package.json:16]: ooh okay, my bad

**Review by mohamedisaaq0 — APPROVED** [review]

**Pull request merged**